frontend/src/pages/EnterReceiptPage/EnterReceiptPage.tsx

```diff
@@ -20,7 +20,10 @@ const EnterReceiptPage: React.FC = () => {
 
   return (
     <div className="tablet:margin-x-8 overflow-x-auto">
-      <Title>Enter Receipt</Title>
+      <div className="grid-row flex-align-start flex-justify">
+        <Title>Enter Receipt</Title>
+        <EnterReceiptStatusIndicator {...submitAllLoadingStatus} />
+      </div>
```

Comment:
Would it look better if the status was below the title (outside this div) and full-width?

Reply:
Yes, based on the meeting with Owen I think that's a good call. Changing to WIP.

Reply:
Well done